tests/s390x_tests/lib/main_common.pm

```diff
@@ -0,0 +1,1909 @@
+            loadtest "x11/sshxterm";
+            loadtest "x11/gnome_control_center";
+            loadtest "x11/gnome_tweak_tool";
+            loadtest "x11/seahorse";
+        }
+    }
+    # shotwell was replaced by gnome-photos in SLE15 & yast_virtualization isn't in SLE15
+    if (is_sle('>=12-sp2') && is_sle('<15')) {
+        loadtest "x11/shotwell/shotwell_import";
+        loadtest "x11/shotwell/shotwell_edit";
+        loadtest "x11/shotwell/shotwell_export";
+        loadtest "virtualization/yast_virtualization";
+        loadtest "virtualization/virtman_view";
+    }
+    if (get_var("DESKTOP") =~ /kde|gnome/) {
+        loadtest "x11/tracker/prep_tracker";
+        # tracker-gui/tracker-needle was dropped since version 1.99.0
+        if (!sle_version_at_least('15')) {
+            loadtest "x11/tracker/tracker_starts";
+            loadtest "x11/tracker/tracker_searchall";
+            loadtest "x11/tracker/tracker_pref_starts";
+            loadtest "x11/tracker/tracker_open_apps";
+            loadtest "x11/tracker/tracker_mainmenu";
+        }
+        loadtest "x11/tracker/tracker_by_command";
+        loadtest "x11/tracker/tracker_info";
+        loadtest "x11/tracker/tracker_search_in_nautilus";
+        loadtest "x11/tracker/clean_tracker";
+    }
+}
+
+sub load_common_x11 {
+    if (check_var("REGRESSION", "installation")) {
+        load_x11_installation;
+    }
+    elsif (check_var("REGRESSION", "gnome")) {
+        loadtest "boot/boot_to_desktop";
+        loadtest "x11/window_system";
+        load_x11_gnome();
+    }
+    elsif (check_var("REGRESSION", "documentation")) {
+        loadtest "boot/boot_to_desktop";
+        loadtest "x11/window_system";
+        load_x11_documentation();
+    }
+    elsif (check_var("REGRESSION", "other")) {
+        loadtest "boot/boot_to_desktop";
+        loadtest "x11/window_system";
+        load_x11_other();
+    }
+}
+
+# Move fips testsuites to main_common to apply to SLE_FIPS + openSUSE
+# Rename load_fips_tests_* to load_security_tests_* to avoid confusedness since
+# openSUSE does NOT have FIPS mode
+# Some tests are only valid for SLE FIPS and loaded if defined variables set
+sub load_security_tests_core {
+    if (check_var('DISTRI', 'sle') && get_var('FIPS_TS')) {
+        loadtest "fips/openssl/openssl_fips_alglist";
+        loadtest "fips/openssl/openssl_fips_hash";
+        loadtest "fips/openssl/openssl_fips_cipher";
+        loadtest "fips/openssh/openssh_fips";
+    }
+    loadtest "fips/openssl/openssl_pubkey_rsa";
+    loadtest "fips/openssl/openssl_pubkey_dsa";
+    if (is_sle('12-SP2+') || is_tumbleweed) {
+        loadtest "console/openssl_alpn";
+    }
+    loadtest "console/sshd";
+    loadtest "console/ssh_pubkey";
+    loadtest "console/ssh_cleanup";
+    loadtest "console/openvswitch_ssl";
+    loadtest "console/consoletest_finish";
+}
+
+sub load_security_tests_web {
+    loadtest "console/curl_https";
+    loadtest "console/wget_https";
+    loadte
```

Comment:
This main_common.pm should be in `tests/sle/lib/` and not in s390x subfolder

Reply:
Yep, correct. We've just put main_common.pm to this folder temporarily as we change only one function in `main_common.pm`. We suppose that you can extract only this function 
`sub load_consoletests` and put it into the master branch main_common.pm

But we can put our versions of both `main_common.pm` and `s390base.pm` into their **real** folders so you can try merging our changes into master automatically? is it ok?

Reply:
> But we can put our versions of both `main_common.pm` and `s390base.pm` into their **real** folders so you can try merging our changes into master automatically? is it ok?

That's the most stable solution I guess

Reply:
Ok, we've moved all the test shell scripts and consoletest_ perl modules to the appopriate locations. Now they are exactly the same as on our test servers where testcases are currently executed.

Here are the shellscripts and lib files and config files: https://github.com/elifaln/os-autoinst-distri-opensuse/tree/s390x_tests/data/s390x
Here are the consoletest perl modules: https://github.com/elifaln/os-autoinst-distri-opensuse/tree/s390x_tests/tests/s390x_tests